src/promptflow/tests/sdk_pfs_test/e2etests/test_run_apis.py

```diff
@@ -81,6 +82,22 @@ def test_archive_restore_run(self, pf_client: PFClient, pfs_op: PFSOperations) -
         runs = pfs_op.list_runs().json
         assert any([item["name"] == run.name for item in runs])
 
+    def test_delete_run(self, pf_client: PFClient, pfs_op: PFSOperations) -> None:
+        run = create_run_against_multi_line_data(pf_client)
```

Comment:
may use a more dummy run

Reply:
This is a very dummy run. this is print env...